src/backend/cuda/Array.hpp

```diff
@@ -36,6 +36,10 @@ void evalNodes(std::vector<Param<T>> &out, std::vector<common::Node *> nodes);
 template<typename T>
 void evalMultiple(std::vector<Array<T> *> arrays);
 
+template<typename T, typename Ti, typename FUNC>
+Array<T> createNodeArray(const dim4 &odims, FUNC createNode,
+                         const Array<Ti> *children, int num_children);
+
```

Comment:
As per our slack discussion, this is just a left over. Just leaving a comment as a reminder to remove this.